lib/RTCSession.js

```diff
@@ -1042,13 +1045,21 @@ module.exports = class RTCSession extends EventEmitter
       }
       else
       {
-        const dtmf = new RTCSession_DTMF(this);
-
-        options.eventHandlers = {
-          onFailed : () => { this._tones = null; }
-        };
-        dtmf.send(tone, options);
-        timeout = duration + interToneGap;
+        switch (type)
+        {
+          case RFC2833:
+            this._getDTMFRTPSender().insertDTMF(tone, duration, interToneGap);
```

Comment:
if the method returns an undefined? `this._getDTMFRTPSender()`

Reply:
Good question, I can handle errors using offensive programing like this:
```
 const sender = this._getDTMFRTPSender();
sender && sender.insertDTMF(tone, duration, interToneGap);
```